prisma/seed.ts

```diff
@@ -5,21 +5,52 @@ import { generateUser, GenerateUserInput } from "../src/utils/recommendation";
 
 const prisma = new PrismaClient();
 
+/**
+ * Deletes all entries in the user table.
+ */
+const deleteUsers = async () => {
+  await prisma.user.deleteMany({});
+};
+
+/**
+ * Clears connections between users.
+ */
+const clearConnections = async () => {
+  const users = await prisma.user.findMany();
+
+  await Promise.all(
+    users.map(async (user) => {
+      return await prisma.user.update({
+        where: {
+          id: user.id,
+        },
+        data: {
+          favorites: {
+            disconnect: new Array(70)
+              .fill(undefined)
+              .map((_, idx) => ({ id: `${idx}` })),
+          },
+        },
+      });
+    })
+  );
+};
+
 /**
  * Generates favorites for users in our database.
  */
 const generateFavorites = async () => {
   const users = await prisma.user.findMany();
 
-  Promise.all(
-    users.map((user, idx) =>
+  await Promise.all(
+    users.map((_, idx) =>
```

Comment:
What's the '_' doing?

Reply:
nothing special, we just never use the user value, so we rename it to _.